### PR TITLE
typo fix with property placeholder

### PR DESCRIPTION
**Pull request by darky**

### CODE REVIEW

File: core/camel-base/src/main/docs/properties-component.adoc [hunk 1]
```diff
@@ -46,7 +46,7 @@ For example:
 
 [source]
 ----
-location=file:${karaf.home}/etc/foo.properties
+location=file:{{karaf.home}}/etc/foo.properties
```
Comment (davsclaus) [inline core/camel-base/src/main/docs/properties-component.adoc:49]: It should be `{{sys:karaf.home}}`

File: core/camel-base/src/main/docs/properties-component.adoc [hunk 2]
```diff
@@ -113,7 +113,7 @@ For fine grained configuration of the location, then this can be done as follows
       optional = "false"/>
     <propertiesLocation
       resolver = "file"
-      path     = "${karaf.home}/etc/my-override.properties"
+      path     = "{{karaf.home}}/etc/my-override.properties"
```
Comment (davsclaus) [inline core/camel-base/src/main/docs/properties-component.adoc:116]: It should be `{{sys:karaf.home}}`

**Review by davsclaus — CHANGES_REQUESTED** [review]

File: core/camel-base/src/main/docs/properties-component.adoc (hunk 2 above)
Comment (darky) [reply core/camel-base/src/main/docs/properties-component.adoc]: ✅

File: core/camel-base/src/main/docs/properties-component.adoc (hunk 1 above)
Comment (darky) [reply core/camel-base/src/main/docs/properties-component.adoc]: ✅

**Review by davsclaus — APPROVED** [review]

**Pull request merged**